reporter/block_handler.go

```diff
@@ -0,0 +1,174 @@
+package reporter
+
+import (
+	"fmt"
+
+	"github.com/babylonchain/vigilante/types"
```

Comment:
why do we import babylon here?

Reply:
I've left a few types imported from babylon cause they are part of checkpointing logic which is to be removed, so can test it for now without too much change to the forked code and remove it at the end of testing.